lib/resources/Topups.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 
 var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
 
 module.exports = StripeResource.extend({
   path: 'topups',
   includeBasic: ['create', 'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'],
+
+  cancel: stripeMethod({
+    method: 'POST',
+    path: '{topupId}/cancel',
```

Comment:
The test seem to confirm it works but other files have a leading `/` in the path.

Reply:
Hm, we seem to be a little inconsistent on this right now — looking through other examples in the library, it seems that both work.